jhipster-framework/src/main/java/io/github/jhipster/config/metric/JHipsterMetricsEndpointConfiguration.java

```diff
@@ -1,26 +0,0 @@
-package io.github.jhipster.config.metric;
```

Comment:
Same for this one, this is required for micrometer.

Reply:
Same

Reply:
I added back files